.htaccess

```diff
@@ -48,7 +48,7 @@ deny from env=stayout
     RewriteRule src/Backend/Core/Js/ckfinder/core/connector/php/connector\.php - [L]
 
 	# forbidden folders
-    RewriteRule .*\.php - [F]
+        RewriteRule .*\.php - [F]
```

Comment:
Maybe the plus made it look like that
<img width="305" alt="screen shot 2017-08-23 at 09 10 07" src="https://user-images.githubusercontent.com/3634654/29603039-dff3c298-87e2-11e7-8f6b-c0fe7a445c81.png">